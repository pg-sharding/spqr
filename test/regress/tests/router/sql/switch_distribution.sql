\c regress
SET __spqr__distribution = ds1;

\c spqr-console
DROP DISTRIBUTION ALL CASCADE;
DROP KEY RANGE ALL;