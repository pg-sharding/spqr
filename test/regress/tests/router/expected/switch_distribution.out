\c regress
SET __spqr__distribution = ds1;
ERROR:  client processing error: setting "__spqr__distribution" is forbidden., tx status IDLE
\c spqr-console

		SPQR router admin console
	Here you can configure your routing rules
------------------------------------------------
	You can find documentation here 
https://github.com/pg-sharding/spqr/tree/master/docs

DROP DISTRIBUTION ALL CASCADE;
 drop distribution 
-------------------
(0 rows)

DROP KEY RANGE ALL;
 drop key range 
----------------
(0 rows)

